netlify/functions/session.js

```diff
@@ -10,17 +11,17 @@ exports.handler = async function () {
   // In case of Daily provider flow, the OpenTok instance
   // will be created by passing in a Daily API Key
   // and empty string.
-  const useDaily = process.env.USE_DAILY;
   if (useDaily === 'TRUE') {
     initParam1 = process.env.DAILY_API_KEY;
     initParam2 = '';
   }
+
   const ot = new OpenTok(initParam1, initParam2);
   let session;
 
   try {
     session = await new Promise((resolve, reject) => {
-      ot.createSession({ mediaMode: 'relayed' }, (error, sess) => {
+      ot.createSession({ mediaMode: 'routed' }, (error, sess) => {
```

Comment:
Just curious, how come we're changing the default here?

Reply:
You're right, I must have changed this file by accident